smart-home-2020-master/src/main/java/ru/sbt/mipt/oop/Application.java

```diff
@@ -0,0 +1,42 @@
+package ru.sbt.mipt.oop;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+public class Application {
+    private final EventCreator eventCreator;
+    private SmartHome smartHome;
+
+
+    public Application(EventCreator eventCreator, SmartHome smartHome) {
+        this.eventCreator = eventCreator;
+        this.smartHome = smartHome;
+    }
+
+    public void run(){
+        SensorEvent event = eventCreator.getNextEvent();
+        ProcessEvent processLight, processDoors, processHallDoors;
+        processLight = new LightProcessEvent();
+        processDoors = new DoorsProcessEvent();
+        processHallDoors = new HallDoorProcessEvent();
+        List<ProcessEvent> processEvents = Arrays.asList(processLight, processDoors, processHallDoors);
```

Comment:
https://github.com/iagoshkov/smart-home-2020/blob/master/STUDENT_GUIDE.md#%D0%BF%D0%B5%D1%80%D0%B5%D0%B4%D0%B0%D0%B2%D0%B0%D0%B9%D1%82%D0%B5-%D0%B7%D0%B0%D0%B2%D0%B8%D1%81%D0%B8%D0%BC%D0%BE%D1%81%D1%82%D0%B8-%D1%81%D0%BD%D0%B0%D1%80%D1%83%D0%B6%D0%B8